gym_solo/envs/test_solo8v2vanilla.py

```diff
@@ -144,5 +146,23 @@ def test_observation_space(self):
     self.env.obs_factory.register_observation(o)
     self.assertEqual(o.observation_space, self.env.observation_space)
 
+  def test_disjoint_environments(self):
+    env1 = solo_env.Solo8VanillaEnv(config=solo_env.Solo8VanillaConfig())
+    env1.obs_factory.register_observation(solo_obs.TorsoIMU(env1.robot))
+    env1.obs_factory.register_observation(solo_obs.MotorEncoder(env1.robot))
+    env1.reward_factory.register_reward(1, SimpleReward())
+
+    home_position = env1.reset()
+    
+    for i in range(1000):
+      env1.step(env1.action_space.sample())
```

Comment:
Why do we need this for loop? There is nothing being tested from env1 except the home position

Reply:
to ensure that `env1` isn't in the home position. then, if `env2`'s observation is the home position, we know that `env1` and `env2` are fully disjoint.